docs/components/FormatRelativeDateTime/Web.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { FormatRelativeDateTime } from "@jobber/components/FormatRelativeDateTime";
+
+export default {
+  title: "Components/Utilities/FormatRelativeDateTime/Web",
+  component: FormatRelativeDateTime,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof FormatRelativeDateTime>;
+
+const BasicTemplate: ComponentStory<typeof FormatRelativeDateTime> = args => (
```

Comment:
The example doesn't seem to work 😅  And we're seeing the empty date in the code

<img width="703" alt="Screenshot 2023-08-01 at 1 50 34 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/56be4e2e-3e62-4cdc-9144-88aa9e1edb33">